### PR TITLE
Move Beam group to Sonar/Beam

### PR DESCRIPTION
**Pull request by emiliom**
Test implementation of #519. Creating a new branch to test this implementation.

Currently two tests are failing (`test_echodata.py::TestEchoData::test_constructor` and `test_echodata.py::TestEchoData::test_repr`) because the test dataset `echopype/test_data/ek60/ncei-wcsd/Summer2017-D20170615-T190214.zarr` will need to be updated to use `Sonar/Beam`.

Not ready to merge. Besides updating that dataset, there's a larger issue about whether our assumptions about Beam groups are correct. See #519.

### CODE REVIEW

**Comment by emiliom** [comment]
All tests now run successfully! To do this I created a new test file converted using the code in this PR, to replace the use of `test_data/ek60/ncei-wcsd/Summer2017-D20170615-T190214.zarr` in `test_echodata.py`. But persisting a new, temporary test file is tricky in the current test data infrastructure. So, I placed the new file, `Summer2017-D20170615-T190214-beamassubgroup.nc`, under `echopype/test_data` for now, and added a `echopype/test_data/.gitignore` so this file can be included in the PR.

Separate note: for the path reference to `Beam` as a subgroup I use `/Sonar/Beam` in most cases and `Sonar/Beam` in a couple of cases (mainly `echopype/echodata/convention/1.0.yml`, I think). I'm not certain that we can't actually use `Sonar/Beam` in all cases. We should check.

File: echopype/tests/convert/test_convert_azfp.py [hunk 1]
```diff
@@ -108,7 +108,7 @@ def test_convert_azfp_01a_different_ranges():
 #     tmp = Convert(file=raw_paths[1:5], model='AZFP', xml_path=raw_paths[0])
 #     tmp.to_netcdf(save_path=export_folder, overwrite=True, combine=True)
 #
-#     with xr.open_dataset(tmp.output_file, group='Beam') as ds_beam:
+#     with xr.open_dataset(tmp.output_file, group='https://washington.zoom.us/j/5615195405?pwd=WG80Y1BGMzVpSWJ4SXplbnFsYXVOZz09Beam') as ds_beam:
```
Comment (emiliom) [inline echopype/tests/convert/test_convert_azfp.py:111]: Yikes, that's a bit wrong! Fortunately it was in a comment!

**Comment by emiliom** [comment]
The CI tests failed; the [new test data file could not be found](https://github.com/OSOceanAcoustics/echopype/runs/4999343470?check_suite_focus=true#step:16:313). But I ran the complete test battery locally (`python .ci_helpers/run-test.py --local --pytest-args="-vv"`), and it completed w/o failures.

**Comment by lsetiawan** [comment]
> The CI tests failed; the new test data file could not be found. But I ran the complete test battery locally (python .ci_helpers/run-test.py --local --pytest-args="-vv"), and it completed w/o failures.

Have you placed your test files in the CI_test_data folder within the google drive?

**Comment by emiliom** [comment]
> Have you placed your test files in the CI_test_data folder within the google drive?

I had not, until 5 minutes ago :smiley: . I wanted to have a persistent-ish way to run the tests (and confirm that my changes worked), so that was my workaround. But now that I'm satisfied my code changes work, I realized that I could add the file to the Google Drive, wait for the dockerhub image update, then redeploy/rebuild the docker image locally whenever I get back to it (not today).

**Comment by emiliom** [comment]
Alright. The file I uploaded to the Google Drive folder was incorporated into today's docker image update. I updated `test_echodata.py` to use that file instead of the half-baked approach I used earlier. :crossed_fingers:

File: echopype/tests/convert/test_convert_azfp.py (hunk 1 above)
Comment (leewujung) [reply echopype/tests/convert/test_convert_azfp.py]: I smell traces of multitasking in here!  :D

Comment (emiliom) [reply echopype/tests/convert/test_convert_azfp.py]: Oh, just a bit :rofl: . Losing track of my clipboard

**Pull request closed without merging**